pkgs/applications/misc/blender/default.nix

```diff
@@ -95,6 +99,8 @@ let
     url = "https://developer.download.nvidia.com/redist/optix/v7.3/OptiX-7.3.0-Include.zip";
     hash = "sha256-aMrp0Uff4c3ICRn4S6zedf6Q4Mc0/duBhKwKgYgMXVU=";
   };
+  BLENDER_EXE =
+    if stdenv.isDarwin then "Applications/Blender.app/Contents/MacOS/Blender" else "bin/blender";
```

Comment:
The `zoom-us` package uses `makeWrapper` in the `postFixup` phase to create `bin/zoom` (see [`…/zoom-us/default.nix:155`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/zoom-us/default.nix#L155) which will also help with nix programs (like `nix run`) that use `meta.mainProgram`. What are your thoughts on this, @gador ?

Reply:
thats a neat idea! I'll look into it!